packages/twenty-server/src/modules/workflow/common/workflow-common.workspace-service.ts

```diff
@@ -35,7 +35,7 @@ export class WorkflowCommonWorkspaceService {
       );
     }
 
-    if (!workflowVersion.trigger || !workflowVersion.trigger?.type) {
+    if (!workflowVersion.trigger) {
```

Comment:
why not using your `assertWorkflowVersionIsValid` utils here?

Reply:
if you only need to check the trigger, maybe you can update `assertWorkflowVersionIsValid` to return after the trigger check

Reply:
I don't mind if the user stores a version with almost nothing in it. Maybe we'll even have an autosave for this so he does not lose its work. So the trigger could be totally empty, that's fine. The only important thing in my opinion is to validate before publishing the version. Does it make sense @martmull?